templates/default/node.yaml.erb

```diff
@@ -1,5 +1,9 @@
 allowDisabledDocker: false
 apiVersion: v1
+<% if @ose_major_version.split('.')[1].to_i >= 6 %>
+dnsBindAddress: 127.0.0.1:53
+dnsRecursiveResolvConf: /etc/origin/node/resolv.conf
```

Comment:
I had to comment those 2 lines, which conflicted with installation when `node['cookbook-openshift3']['use_dnsmasq']` is truthy. I don't know where these 2 attributes come from but:
 - commenting them out allows me to provision my VM successfully
 - the 2 attributes are not documented in https://docs.openshift.com/container-platform/3.6/install_config/master_node_configuration.html

Reply:
@jperville I have put a quick logic for matching the default GW and that should work better for you.

Reply:
Thanks @IshentRas , testing now... I found that with the above 2 lines commented I can install openshift, but the internal kube-dns is not working. Will let you know how it goes

Reply:
:+1: